### PR TITLE
Feature/implement error handlers

### PR DESCRIPTION
**Pull request by giovanneguerra**
Create error handling functions

### CODE REVIEW

File: apps/explorer/nuxt.config.ts [hunk 1]
```diff
@@ -2,6 +2,10 @@ import { defineNuxtConfig } from 'nuxt/config'
 
 export default defineNuxtConfig({
   modules: ['@nuxtjs/tailwindcss', '@nuxtjs/color-mode', 'nuxt-graphql-client'],
+  components: {
+    global: true,
+    dirs: ['~/components']
+  },
```
Comment (smakhtin) [inline apps/explorer/nuxt.config.ts:8]: Why is this modification needed? I think in Nuxt 3 all components are global by default.

File: apps/explorer/error.vue [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+      </button>
+    </div>
+  </div>
+</template>
+<script setup>
+defineProps({
+  error: {
+    type: Object,
+    required: false,
+    default: () => Object
```
Comment (smakhtin) [inline apps/explorer/error.vue:26]: Maybe we can write a default error object here?

**Review by smakhtin — CHANGES_REQUESTED** [review]
Looks good, have some small questions.

File: apps/explorer/nuxt.config.ts (hunk 1 above)
Comment (giovanneguerra) [reply apps/explorer/nuxt.config.ts:8]: It isn't or else it doesn't let you change the variable state in the template

**Pull request merged**